src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react"
+import { getUserByUsername } from "../api"
+
+export function ArticleCard ({article}) {
+
+    const [avatarURL, setAvatarURL] = useState("https://www.refugee-action.org.uk/wp-content/uploads/2016/10/anonymous-user.png")
+
+    useEffect(()=>{
+        getUserByUsername(article.author).then(({avatar_url})=>{
```

Comment:
I really like the idea here, at the moment this is making a request per article, which is a lot, I would leave it for now but consider changing the backend (at a later date) to include the avatar on the article request to achieve what you want, this way you'd only need the one request and it would be more efficient.